ibm_cloud_sdk_core/utils.py

```diff
@@ -22,6 +23,9 @@
 
 import dateutil.parser as date_parser
 
+if 'VCAP_SERVICES_FILE' not in os.environ.keys():
```

Comment:
I don't think we want to support an env variable that specifies the name of a file containing the VCAP object.
The only thing we currently support is the VCAP_SERVICES environment variable that contains the actual VCAP object (JSON string).
On the surface, adding support for loading the VCAP object from a file might be a good thing, but:
1) We really don't want to encourage anyone to use VCAP_SERVICES if they aren't already using it (it's sort of "stale").
2) We'd need to replicate this new functionality across all the cores.